### PR TITLE
Added analytics data collection for the PID loop frequency.

### PR DESCRIPTION
**Pull request by mikeller**

### CODE REVIEW

File: src/js/tabs/configuration.js [hunk 1]
```diff
@@ -1235,7 +1235,17 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             PID_ADVANCED_CONFIG.use_unsyncedPwm = $('input[id="unsyncedPWMSwitch"]').is(':checked') ? 1 : 0;
             PID_ADVANCED_CONFIG.motor_pwm_rate = parseInt($('input[name="unsyncedpwmfreq"]').val());
             PID_ADVANCED_CONFIG.gyro_sync_denom = parseInt(gyroSelectElement.val());
-            PID_ADVANCED_CONFIG.pid_process_denom = parseInt(pidSelectElement.val());
+
+            const value = parseInt(pidSelectElement.val());
+
+            let newValue = undefined;
```
Comment (McGiverGim) [inline src/js/tabs/configuration.js:1241]: Maybe is better to give the variables a name more descriptive, this are too generic.

**Comment by etracer65** [comment]
From slack comment:

Not sure you want to capture just the actual frequency - at least not without capturing the gyro sample rate to go along with it. With support for base frequencies besides 8K the final frequency is out of context without knowing the `pid_process_denom` and base frequency. Like a bmi160 running with `pid_process_denom = 4` (800hz) is the same as a MPU6000 with `pid_process_denom = 10`.

File: src/js/tabs/configuration.js (hunk 1 above)
Comment (mikeller) [reply src/js/tabs/configuration.js:1241]: One thing I want to do for 10.8 is to refactor and centralise these analytics helper functions, now that we have well established patterns.

**Comment by mikeller** [comment]
Added analytics tracking for the PID loop denominator as per @etracer65's request.

**Pull request merged**